decorator.rb

```diff
@@ -0,0 +1,11 @@
+require './nameable'
+
+class Decorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable
+  end
```

Comment:
- [ ] Already initialized in the constructor it should be called @nameable.correct_name